src/plugins/visualizations/public/embeddable/visualize_embeddable.ts

```diff
@@ -249,6 +253,12 @@ export class VisualizeEmbeddable
       this.updateOutput({ title: this.vis.title });
     }
 
+    // propagate the placeholder title to the output embeddable
+    // but only when the visualization is in edit/Visualize mode
+    if (!this.parent && this.placeholderTitle !== this.output.placeholderTitle) {
+      this.updateOutput({ placeholderTitle: this.placeholderTitle });
```

Comment:
Not sure what you mean by "every embeddable", but if you mean "every visualize embeddable that had its placeholder title changed", then the answer is yes 😄

Reply:
Oh I see, I thought that this was required to propagate the dashboard placeholder title to the embeddable output so that the panel header could pick it up. It sounds like this is to support custom overrides to the placeholder title by embeddables?